functions/helpers.bash

```diff
@@ -53,6 +53,12 @@ is_pione() {
     return 1
   fi
 }
+is_cmone() {
```

Comment:
can you please comment somewhere in the file?
https://www.raspberrypi.org/documentation/hardware/raspberrypi/revision-codes/README.md